api/utils.py

```diff
@@ -337,17 +332,12 @@ async def put(
             user: Union[None, ModelView.schemes.User] = Security(self.auth_dependency, scopes=self.scopes["put"]),
         ):  # type: ignore
             item = await self._get_one(model_id, user, True)
-            try:
+            with safe_db_write():
+
```

Comment:
Just a question: was this space added by black or not? If not, remove it please

Reply:
good catch! maybe it's me